src/client/src/components/TopNavBarLink/styles.module.css

```diff
@@ -21,7 +21,7 @@
   display: flex;
   align-items: center;
   text-decoration: none;
-  height: 52px;
```

Comment:
I'm comparing this PR to the one that's already merged, and the height is not the same as that
https://github.com/microsoft/WebTemplateStudio/pull/860/files#diff-139cb5b75723e18fb784ac672b5273b0L24

Reply:
its fine